client/client.go

```diff
@@ -0,0 +1,150 @@
+package client
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"time"
+
+	"github.com/google/netstack/tcpip"
+	"github.com/google/netstack/tcpip/header"
+	"github.com/u-root/dhcp6"
```

Comment:
All imports need to remain as mdlayher/dhcp6.  I find it's easiest to `go get` this to the usual spot and then just add your fork as a remote to push to.